### ISSUE TITLE
VSCodeInsider_VSOnline_Extension_VisualStudioOnline_ScreenReader:Narrator does not announces anything when "***** receives screen reader focus.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#88959]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
OS Version: Win10

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Navigate  right panel till "Extension: Visual Studio Online" 
4. start the Narrator and navigate on the screen using Caps+right arrow key.
5. Observe Narrator does not announces anything when below control receives the screen reader focus.
![image](https://user-images.githubusercontent.com/55909580/72716099-a99d9a00-3b97-11ea-9f16-48ae102057e9.png)

**Actual:**
Narrator does not announces anything when above mentioned  control receives the screen reader focus.

**Expected:​**
Narrator should  announces related information or name  when above mentioned control receives the screen reader focus.

**​User Impact:​**
Screen reader does not understand about the control if it does not announces.
[88959_VSCodeInsider_VSOnline_Extension_VisualStudioOnline_ScreenReader.pptx](https://github.com/microsoft/vscode/files/4085518/88959_VSCodeInsider_VSOnline_Extension_VisualStudioOnline_ScreenReader.pptx)

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/88792
